src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -86,11 +86,11 @@ function SubscriptionManagementPage({ enterpriseSlug, enterpriseId }) {
                         </div>
                         <div className="col-12 col-lg-7">
                           <AddUsersButton
-                            onSuccess={() => {
+                            onSuccess={(response) => {
```

Comment:
Do we want to display any of the error response from the endpoint in the admin portal?

Reply:
That's outside the scope of this ticket, for now we're going with the success message that indicates already assigned and newly assigned. Updating the UI for 400 responses would have to be a different ticket

Reply:
As is, we should be covered with displaying an error alert with the `error.message` that comes back from the API call within the `AddUsersModal` (https://github.com/edx/frontend-app-admin-portal/blob/master/src/components/AddUsersModal/index.jsx#L117). Of course, if that message could be cleaned up or more useful, it probably makes sense to do in another ticket so we can get more input from Product/UX in terms of copy, etc.

Reply:
Yeah I know it wouldn't be hard; we had a pretty lengthy conversation about error handling in addition to what is covered by the AC for this ticket and it was decided that nothing was going to be done in this ticket with errors, only the success message. I'm not expanding the scope of this ticket to include something we directly talked about when grooming it